src/demux/readers/eos/MongoActionReader.ts

```diff
@@ -0,0 +1,65 @@
+import { AbstractActionReader } from "../AbstractActionReader"
+import { MongoBlock } from "./MongoBlock"
+
+import { MongoClient } from "mongodb"
+
+/**
+ * Implementation of an ActionReader that polls a mongodb.
+ */
+export class MongoActionReader extends AbstractActionReader {
+  protected isInitialized: boolean
```

Comment:
This can go in the constructor declaration and have the default to `false`

Reply:
If this goes in the constructor the user can set this by passing in a value right? If thats the case then it doesn't seem like the behavior we want, the MongoActionReader should control that variable.